cmd/seed-controller-manager/options.go

```diff
@@ -128,6 +129,7 @@ func newControllerRunOptions() (controllerRunOptions, error) {
 	flag.StringVar(&c.backupContainerImage, "backup-container-init-image", backupcontroller.DefaultBackupContainerImage, "Docker image to use for the init container in the backup job, must be an etcd v3 image. Only set this if your cluster can not use the public quay.io registry")
 	flag.StringVar(&c.backupInterval, "backup-interval", backupcontroller.DefaultBackupInterval, "Interval in which the etcd gets backed up")
 	flag.StringVar(&rawEtcdDiskSize, "etcd-disk-size", "5Gi", "Size for the etcd PV's. Only applies to new clusters.")
+	flag.IntVar(&c.etcdReplicasNumber, "etcd-replicas-number", 3, "Number of etcd replicas. Only applies to new clusters.")
```

Comment:
Can we use `kubermaticv1.DefaultEtcdClusterSize` instead of the hardcoded 3 here?